src/Symfony/Bridge/Doctrine/Messenger/EventSubscriber/EntityMessageCollector.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger\EventSubscriber;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Events;
+use Symfony\Bridge\Doctrine\Messenger\EntityMessage\EntityMessageCollectionInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\DispatchAfterCurrentBusStamp;
+
+/**
+ * Doctrine listener that listens to Persist, Update and Remove. Every time this is
+ * invoked we take messages from the entities.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Matthias Noback <matthiasnoback@gmail.com>
```

Comment:
is this based on some existing code? then it should be referenced

Reply:
How would such reference look?

https://github.com/SimpleBus/SimpleBus/blob/master/Bridge/DoctrineORMBridge/src/EventListener/CollectsEventsFromEntities.php

Reply:
```
 * @see https://github.com/SimpleBus/SimpleBus/blob/master/Bridge/DoctrineORMBridge/src/EventListener/CollectsEventsFromEntities.php
```

?